### PR TITLE
[riscv64-generic] Rebase on latest v6.1.106

### PR DESCRIPTION
**Pull request by rouming**
Just `git rebase v6.1.106`.

### CODE REVIEW

**Review by rucoder — APPROVED** [review]
LGTM. Approve to kick-off the build

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+name: eve-kernel build
+
+on:
+  workflow_dispatch:
+  pull_request_review:
+    types: [ submitted ]
+  push:
+    branches:
+      - "eve-kernel-riscv64-v6.1.38-generic"
```
Comment (rucoder) [inline .github/workflows/publish.yml]: Change branch name!

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  packages:
+    runs-on: self-hosted
+    if: ${{ github.event.review.state == 'approved' }} || github.ref == env.branch_name
+    steps:
+      - name: Get eve-kernel
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.head_ref }}
+
+      - name: Build eve-kernel-riscv64
```
Comment (rucoder) [inline .github/workflows/publish.yml:27]: the same

File: .github/workflows/publish.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+          make -f Makefile.eve BRANCH?=${GITHUB_REF##*/} kernel-gcc
+
+      - name: Log in to Docker Hub
+        if: ${{ github.ref == env.branch_name }}
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.RELEASE_DOCKERHUB_ACCOUNT }}
+          password: ${{ secrets.RELEASE_DOCKERHUB_TOKEN }}
+      
+      - name: Push eve-kernel-riscv64-v6.1.38-generic if PR approved or pushed
```
Comment (rucoder) [inline .github/workflows/publish.yml]: the same

**Review by rucoder — CHANGES_REQUESTED** [review]

File: .github/workflows/publish.yml (hunk 1 above)
Comment (rouming) [reply .github/workflows/publish.yml]: Done

File: .github/workflows/publish.yml (hunk 2 above)
Comment (rouming) [reply .github/workflows/publish.yml:27]: Done

File: .github/workflows/publish.yml (hunk 3 above)
Comment (rouming) [reply .github/workflows/publish.yml]: Done

**Review by rucoder — APPROVED** [review]
LGTM

**Pull request merged**